lib/sierra-models/base.js

```diff
@@ -160,12 +229,29 @@ class SierraBase {
     return this._vars().filter((f) => f.marcTag === `${marc}`)
   }
 
-  // returns array of objects. value is concatenation of contents of
-  //   specified subfields - {value: 'val parallel link', subfieldMap: {a:'val', 6:'parallel link'}}
+  /**
+   *  @typedef VarFieldMatch
+   *  @type {object}
+   *  @property {string} value - The primary value built from joining the subfield values
+   *  @property {object} subfieldMap - An object that relates each subfield to it's value(s). Multiple values for a single subfield will be given as an array
+   *  @property {VarFieldMatchParallel} parallel
+   *
+   *  @typedef VarFieldMatchParallel
+   *  @type {object}
+   *  @property {string} value - The parallel value built by joining the subfield values
+   *  @property {object} subfieldMap - An object that relates each subfield to it's value(s). Multiple values for a single subfield will be given as an array
+   *  @property {string} script - Identifies the script class. One of arabic, latin, cjk, cyrillic, greek
+   *  @property {string} direction - Identifies the script direction. One of rtl, ltr
+   *
+   *
+   * Given an array of raw varField entries from the Sierra record and the
+   * queries sufields and options, returns VarFieldMatch objects
+   *
+   * @return {VarFieldMatch[]}
+   */
   _convertRawFieldToVarFieldMatch (rawFields, includedSubfields, opts) {
     opts = opts || {}
     opts = Object.assign({
-      tagSubfields: false,
       subfieldJoiner: ' ',
```

Comment:
we don't have any subfieldJoiner properties in the mappings anymore. It looks like it was just for the subjectLiteral. I'm happy to see this option workflow be ~~eliminated~~ trimmed down as much as possible. Can we remove the subfieldJoiner logic that remains? I'm also curious if we're using the preFilter option much/at all anymore and if we can get rid of that too?